test/kubernetes/e2e/defaults/testdata/http_echo.yaml

```
@@ -26,9 +27,9 @@ metadata:
 spec:
   containers:
   - name: http-echo
-    image: kennship/http-echo@sha256:144322e8e96be2be6675dcf6e3ee15697c5d052d14d240e8914871a2a83990af
+    image: jmalloc/echo-server
```

we should probably use an image checksum here to be safe.
```suggestion
    image: jmalloc/echo-server@sha256:c87f80de8dbb976a59b228fc9ecf257e0574c6f760c2f1c5f05f64c7ac7fbc1e
```